app/controllers/benchmark_runs_controller.rb

```diff
@@ -31,6 +31,12 @@ def create
     benchmark_run.result = params[:benchmark_run][:result]
     benchmark_run.save!
 
+    if params[:commit_hash]
+      BenchmarkRegressionJob.new.perform(commit_hash: params[:commit_hash], category: benchmark_type_params[:category],
+      script_url: benchmark_type_params[:script_url], name: benchmark_result_type_params[:name],
+      unit: benchmark_result_type_params[:unit], repos: params[:repo], organizations: params[:organization] )
+    end
```

Comment:
You already have the `benchmark_run` and the other records. Just pass in their ID and do a `find(id)` in the job. 

Not sure if that wasn't clear in https://github.com/mperham/sidekiq/wiki/Best-Practices#1-make-your-job-parameters-small-and-simple